### PR TITLE
Hosting Metrics: Add tooltips to each of our charts

### PR DESCRIPTION
**Pull request by danielbachhuber**
See https://github.com/Automattic/dotcom-forge/issues/3276
See https://github.com/Automattic/dotcom-forge/issues/3274
See https://github.com/Automattic/dotcom-forge/issues/3356

## Proposed Changes

Adds tooltips to each of our charts:

CleanShot 2023-08-09 at 05 10 14@2x

CleanShot 2023-08-09 at 05 10 27@2x

CleanShot 2023-08-09 at 05 10 36@2x

CleanShot 2023-08-09 at 05 10 47@2x

## Testing Instructions

1. Navigate to `/site-monitoring/<site>`
2. Verify all of the tooltips appear as expected.

### CODE REVIEW

**Comment by danielbachhuber** [comment]
@javierarce Can you annotate the diff with the text you'd like for each tooltip? They don't seem to be fully documented in the Figma.

Also, do you want the tooltips to appear on click or on hover?

**Comment by javierarce** [comment]
> @javierarce Can you annotate the diff with the text you'd like for each tooltip? They don't seem to be fully documented in the Figma.

Sure, I'll add them here too.

> Also, do you want the tooltips to appear on click or on hover?

On click, since that's what we are doing in other sections in Calypso.

File: client/my-sites/site-monitoring/metrics-tab.tsx [hunk 1]
```diff
@@ -152,19 +152,28 @@ export const MetricsTab = () => {
 			<TimeDateChartControls onTimeRangeChange={ handleTimeRangeChange }></TimeDateChartControls>
 			<SiteMonitoringLineChart
 				title={ __( 'Requests per minute & average response time' ) }
+				tooltip={ __(
+					'Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum.'
```
Comment (javierarce) [inline client/my-sites/site-monitoring/metrics-tab.tsx]: **Requests per minute**: a line representing the number of requests received every minute.

 **Average Response Time**: a line that indicates the average time taken to respond to a request within that minute.

File: client/my-sites/site-monitoring/metrics-tab.tsx [hunk 2]
```diff
@@ -152,19 +152,28 @@ export const MetricsTab = () => {
...
 					data={ getFormattedDataForPieChart( cacheHitMissFormattedData, {
 						0: 'Cache miss',
 						1: 'Cache hit',
 					} ) }
 				></SiteMonitoringPieChart>
 				<SiteMonitoringPieChart
-					title="PHP vs. static content served"
+					title={ __( 'PHP vs. static content served' ) }
+					tooltip={ __(
+						'Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum.'
```
Comment (javierarce) [inline client/my-sites/site-monitoring/metrics-tab.tsx]: Percentages showing the ratio of dynamic versus static content. Dynamic content is the content generated using database information.

File: client/my-sites/site-monitoring/metrics-tab.tsx [hunk 3]
```diff
@@ -173,7 +182,10 @@ export const MetricsTab = () => {
 				></SiteMonitoringPieChart>
 			</div>
 			<SiteMonitoringBarChart
-				title={ __( 'Requests by HTTP Response Code' ) }
+				title={ __( 'Requests by HTTP response code' ) }
+				tooltip={ __(
+					'Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum.'
```
Comment (javierarce) [inline client/my-sites/site-monitoring/metrics-tab.tsx]: Number of requests categorized by their HTTP response code. Hover over each entry in the legend for detailed information.

**Comment by javierarce** [comment]
I added the texts in the diffs and left the design for all the tooltips here 7LKer2QaciNxdOgX7O9oqP-fi-327_5085:

image

**Comment by danielbachhuber** [comment]
@javierarce Works great, thanks!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8693836

Thank you @danielbachhuber for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8693836).

**Pull request merged**